capstone/src/main/webapp/style.css

```diff
@@ -2,3 +2,46 @@ body {
   background-color: white;
   margin: 0;
 }
+
+/*
+ * Code source from W3Schools: https://www.w3schools.com/howto/howto_js_autocomplete.asp
+ */
+.autocomplete {
+  position: relative;
```

Comment:
Looks like this didn't get addressed: https://github.com/googleinterns/univiz/pull/50#discussion_r478573816

Reply:
sorry-didn't see the new comments! I hope it is addressed now!